diff --git a/node_modules/tua-body-scroll-lock/dist/tua-bsl.esm.browser.js b/node_modules/tua-body-scroll-lock/dist/tua-bsl.esm.browser.js
index 1e3a6a8..86d5d12 100644
--- a/node_modules/tua-body-scroll-lock/dist/tua-bsl.esm.browser.js
+++ b/node_modules/tua-body-scroll-lock/dist/tua-bsl.esm.browser.js
@@ -50,7 +50,7 @@ const lockedElements = [];
 const eventListenerOptions = getEventListenerOptions({ passive: false });
 const supportsNativeSmoothScroll = !isServer() && 'scrollBehavior' in document.documentElement.style;
 const setOverflowHiddenPc = () => {
-    const $body = document.body;
+    const $body = document.documentElement;
     const bodyStyle = Object.assign({}, $body.style);
     const scrollBarWidth = window.innerWidth - $body.clientWidth;
     $body.style.overflow = 'hidden';
diff --git a/node_modules/tua-body-scroll-lock/dist/tua-bsl.esm.js b/node_modules/tua-body-scroll-lock/dist/tua-bsl.esm.js
index 5cd4e1c..caf2d50 100644
--- a/node_modules/tua-body-scroll-lock/dist/tua-bsl.esm.js
+++ b/node_modules/tua-body-scroll-lock/dist/tua-bsl.esm.js
@@ -59,7 +59,7 @@ var eventListenerOptions = getEventListenerOptions({
 var supportsNativeSmoothScroll = !isServer() && 'scrollBehavior' in document.documentElement.style;

 var setOverflowHiddenPc = function setOverflowHiddenPc() {
-  var $body = document.body;
+  var $body = document.documentElement;
   var bodyStyle = Object.assign({}, $body.style);
   var scrollBarWidth = window.innerWidth - $body.clientWidth;
   $body.style.overflow = 'hidden';
diff --git a/node_modules/tua-body-scroll-lock/src/index.ts b/node_modules/tua-body-scroll-lock/src/index.ts
index aaa693f..a9eb42f 100644
--- a/node_modules/tua-body-scroll-lock/src/index.ts
+++ b/node_modules/tua-body-scroll-lock/src/index.ts
@@ -19,7 +19,7 @@ const eventListenerOptions = getEventListenerOptions({ passive: false })
 const supportsNativeSmoothScroll = !isServer() && 'scrollBehavior' in document.documentElement.style

 const setOverflowHiddenPc = () => {
-    const $body = document.body
+    const $body = document.documentElement
     const bodyStyle = { ...$body.style }
     const scrollBarWidth = window.innerWidth - $body.clientWidth

